modules/ROOT/pages/deployment/binary/small-scale.adoc

```diff
@@ -0,0 +1,258 @@
+
+[source,bash]
+----
+sudo wget -O /usr/local/bin/ocis <file_url>
+----
+
+Make the binary executable with:
+
+[source,bash]
+----
+sudo chmod +x /usr/local/bin/ocis
+----
+
+Check the version installed:
+
+[source,bash]
+----
+ocis version
+----
+
+The output looks like this:
+
+[source,plaintext,subs="attributes+"]
+----
+ocis version {ocis-version}
+----
+
+== Create service user
+
+Next you need to create a user who will run the service and own all the files of the Infinite Scale service.
+
+[source,bash]
+----
+sudo useradd -r ocis
+----
+
+== Data directory
+
+Then you need to create the data directory. All system data will be stored here and all files uploaded by the users.
+
+[source,bash]
+----
+sudo mkdir /var/lib/ocis
+----
+
+Make the ocis user the owner of the data directory.
+
+[source,bash]
+----
+sudo chown ocis:ocis /var/lib/ocis
+----
+
+== Config file
+
+Now create a directory and config file.
+
+[source,bash]
+----
+sudo mkdir /etc/ocis
+sudo touch /etc/ocis/ocis.env
+----
+
+[source,bash]
+----
+sudo chown -R ocis:ocis /etc/ocis
+----
+
+Then fill the environment file `/etc/ocis/ocis.env` with the following content.
+You need to adjust the `OCIS_URL` variable to hold your domain where Infinite Scale will be reachable. If that is on a port other than `443` you need to also add the port like `https://ocis.example.com:4242`.
+
+[source]
+----
+OCIS_URL=https://ocis.example.com
+PROXY_HTTP_ADDR=0.0.0.0:9200
+PROXY_TLS=false
+OCIS_INSECURE=false
+
+OCIS_LOG_LEVEL=warn
+
+OCIS_CONFIG_DIR=/etc/ocis
+OCIS_BASE_DATA_PATH=/var/lib/ocis
+----
+
+`PROXY_TLS` is set to `false` because TLS termination will be done by nginx.
+
+Next you need to generate the initial oCIS configuration file.
+[source,bash]
+----
+sudo -u ocis ocis init --config-path /etc/ocis
+----
+
+You will be asked if you want to configure Ininite Scale with certificate checking disabled.
+You can safely answer `yes`. oCIS is composed of several microservices which encrypt the communication between them with TLS.
+By answering with `yes` the communication is still encrypted but client and server certificates aren't checked for validity. Usually this is good enough.
+If you want an extra secure deployment you can of course provide your own TLS certificates and activate the certificate checking.
+
+NOTE: The command when successful will show you the initial admin users password. Write it down somewhere safe so that you can log in when the setup is complete.
+
+
+This will have created a configuration file `/etc/ocis/ocis.yaml`.
+
+IMPORTANT: Do NOT delete the  `ocis.yaml` file without creating a backup of it first.
+When you regenerate the `ocis.yaml` file it will be out of sync with the stored user data in the ocis data directory `/var/lib/ocis`.
```

Comment:
hm what piece axactly, only the admin password should be a problem, right? if the issuer changes that might currently be a problem because, although I'm not so sure ... what else?